docs-src/content/cli/Transcribe.md

```diff
@@ -0,0 +1,160 @@
+| -a | `--fmt.timeline.maxAlts` | int | Maximum number of alternatives to provide for each result in the timeline format. (default 1) |
+
+
+### List-file
+
+To process multiple files, use the `--list-file` flag and specify a text file instead of an audio file.
+The text file should be tab-delimited, with one audio file per line.  Each line should begin with a unique
+identifier containing no white space, which will be used to identify which transcript belongs to which 
+input file.  
+
+Example:
+```
+utterance1  /mnt/audio/abcd.wav
+utterance2  /mnt/audio/efgh.wav
+```
+
+When cubic-cli writes to stdout (`-o "-"`), each output will be prefaced by the utterance id and an extra newline. 
+
+### Output
+
+Output must be an existing directory to which the transcripts will be written.
+
+In single-file mode, the transcript will be given the same name as the audio file, with a .txt extension.
+
+In list-file mode, each file processed will have a separate output file, using the utteranceID as the filename with a ".txt" extention.
+
+In either mode, "-" prints to stdout, which is the default behavior.
+
+### Model
+
+A cubicsvr instance may be configured to use one or more models, each with a model id specified in cubicsvr.cfg.toml. The specified id must be one of the ones configured for the cubicsvr instance.
+
+For example, if connecting to the demo server, model=1 specifies the U.S. English 16 kHz model and model=2 specifies the U.S. English 8 kHz model.
+
+The models for a server may be listed by calling `./bin/cubic-cli --insecure --server localhost:2727 models`.
+
+Example:
+```bash
+$ bin/cubic-cli -s demo-cubic.cobaltspeech.com:2727 models
+Model ID '"1"': English (US) 16KHz (Sample Rate: 16000)
+Model ID '"2"': English (US) 8KHz (Sample Rate:  8000)
+Model ID '"3"': Portuguese (BR) 8KHz (Sample Rate:  8000)
+Model ID '"4"': Spanish (ES) 16KHz (Sample Rate: 16000)
+
+$ bin/cubic-cli -s demo-cubic.cobaltspeech.com:2727 transcribe -m 2 sample-8khz.wav 
+```
+
+### Audio channels
+
+By default, cubic-svr assumes the input is a mono file and transcribes channel 0.
+Pass `--audioChannels` to specify other channels.
+
+E.g. `-c "0,2"` would transcribe the first and third channels in the file.
+
+`--stereo` is equivalent to `-c 0,1`
+
+`-c` or `--audioChannel` overrides `--stereo` if both are provided.
+
+### Workers
+
+The `workers` parameter limits the number of requests to send to cubicsvr at once.  
+
+Please note, while this value is defined client-side, the throughput will be limited by 
+the available computational ability of the server.  If you are the only connection to an 
+8-core server, then "-n 8" is a reasonable value. 
+
+### Output format
+
+See [Formats](/sdk-cubic/cli/formats)
+
+## Examples
+
+There are two audio files in the testdata folder.
+They are US English, recorded at 16kHz.
+The filenames contain the intended transcription.
+
+As a quick start, these commands are provided as examples of how to use the
```

Comment:
Which commands?  The ones below?  They are a few paragraphs away from this paragraph.